css/index.css

```diff
@@ -143,4 +143,109 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
+/*--------- Navigation -----------*/
+
+header{
+    display:flex;
+    margin-bottom:30px;
+    margin-top:30px;
+    justify-content: space-between;
+    align-items: center;
+}
+
+nav a{
+        margin-right: 65px;
+        color:black;
+        text-decoration: none;
+}
+
+/*--------- Paragraphs Top -----------*/
+
+.top-content{
+    margin-top:20px;
+    margin-bottom:10px;
+}
+
+/*--------- Boxes -----------*/
+
+/* I chose to color the boxes in HTML because I felt like it just created a big mess
+of CSS code for me. But I realize that in other circumstances that may not be the best
+option when you want to change their style.
```

Comment:
* Try not to use in-line styling in the HTML. What happens is either your future self, or another dev if you were working in a team, would proceed to add code to the CSS, and get frustrated when the background colors don't change on the boxes. The cycle of frustration continues until they realize an in-line style is overriding everything they're doing. In short, it's not a good practice, though it has its uses.

Reply:
You'll find ways over the next few weeks how to CSS code more streamlined and "not a big mess" while keeping styling contained in the CSS.